src/types/ListItem.ts

```diff
@@ -2,4 +2,5 @@ export type ListItem = {
   id: number;
   url: string;
   date: string;
+  status: boolean;
```

Comment:
It might be subjective, but unless the status of the item should really be a boolean value, I'd propose two possible changes with this one.
Either having something like `status: 'read' | 'unread'` or `alreadyRead: boolean`.

Reply:
Perfect, `alreadyRead: boolean` really brings more clarity.